config/context.go

```diff
@@ -0,0 +1,33 @@
+package config
```

Comment:
This file doesn't seemed to be called anywhere, from what I can see? No description of it in the PR also, if it's needed, can you add a description of what this is used for in the commit message/ PR?

Reply:
removed.